src/main/java/org/example/deck/Deck.java

```diff
@@ -16,14 +14,9 @@ public class Deck {
     final String[] symbols = new String[]{"2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A"};
     final Suit[] suits = new Suit[]{Suit.HEART, Suit.CLUB, Suit.SPADE, Suit.DIAMOND};
 
-//    public Deck() {
-//        createFullDeck();
-//    }
-
-
-// return a collection ?? return Collections.unmodifiableList(cards)
-// (unmodifianble is a method provided by the collections class - returns unmodifiable/immutable view of the specified list.
-// once created you canonot modify its content but can access its elements...
+    public Deck() {
+        createFullDeck();
+    }
```

Comment:
I'll need to change my code for it to work with this, is it needed?

Reply:
no! -i had commented it out when working on my game but wasn't sure if you or cheryl needed it so added it back in but will delete!